lib/src/tools/display/display_service_contract.dart

```diff
@@ -0,0 +1,52 @@
+abstract class DisplayService {
+  ///
+  /// Call this method to set brightness for one display
+  ///
+  void setBrightness(DisplayDevice display, double brightness);
+
+  /// Gets whether Display adapter is turn ON.
```

Comment:
Is it proper comment?

Reply:
Changed comment: "Gets whether Display Dark Mode is turn ON.".